src/v/cluster/node/types.cc

```diff
@@ -34,11 +33,10 @@ std::ostream& operator<<(std::ostream& o, const disk& d) {
 std::ostream& operator<<(std::ostream& o, const local_state& s) {
     fmt::print(
       o,
-      "{{redpanda_version: {}, uptime: {}, disks: {}, timestamp: {}}}",
+      "{{redpanda_version: {}, uptime: {}, disks: {}}}",
       s.redpanda_version,
       s.uptime,
-      s.disks,
-      s.timestamp);
+      s.disks);
```

Comment:
this hunk probably needs to be moved to the previous commit. the previous won't compile since this field was removed.